test-requirements.txt

```diff
@@ -1,5 +1,6 @@
 coverage>=4.0.3
 nose>=1.3.7
```

Comment:
OK. this looks good, can you rebase your branch so I can merge it? (the branch is out of date).